gulp/tasks/clean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var del = require( 'del' );
+var config = require( '../config' ).clean;
```

Comment:
Should be `var clean`, see https://nodejs.org/api/child_process.html for example of naming style.